client/src/modules/journal/journal.js

```diff
@@ -340,6 +350,9 @@ function JournalController(Journal, Sorting, Grouping,
         vm.gridOptions.gridFooterTemplate = '<div><strong>' + $translate.instant('FORM.INFO.NUM_TRANSACTION') +
```

Comment:
Would this also be solved if you just made the footer template into:
```html
<div>
  <strong>
    <span translate>FORM.INFO.NUM_TRANSACTION</span>:
    {{ grid.appScope.numberGridTransactions }} / {{ grid.appScope.numberTotalSystemTransctions }}
  </strong>
</div>
```

Then instead of adding in `gridApi`, `$translate`, and other variables in the journal's controller code, you can just write
```js
vm.numberCurrentGridTransactions = records.aggregate.length;
```

Maybe I'm missing something though ...